DOCUMENTATION.md

```diff
@@ -1,810 +1,780 @@
 args = {
-  'older_than' => 'e194efd1-c34d-49c9-9915-e4267e01e92e' # => Notifications::Client::ReceivedText
+  older_than: 'e194efd1-c34d-49c9-9915-e4267e01e92e'
 }
-received_texts = client.get_received_texts(args)
+response = client.get_received_texts(args)
 ```
 
-</details>
-
+To get older messages, pass the ID of an older notification into the `older_than` argument. This will return the next oldest messages from the specified notification ID.
 
-#### Response
+If you leave out the `older_than` argument, the client returns the most recent 250 notifications.
 
-If the request is successful a `Notifications::Client::ReceivedTextCollection` is returned.
+### Arguments
 
-<details>
-<summary>
-Click here to expand for more information.
-</summary>
+#### older_than (optional)
 
-`ReceivedTextCollection` -
+Input the ID of a received text message into this argument. If you use this argument, the client returns the next 250 received text messages older than the given ID.
 
 ```ruby
-received_texts.links # => Hash containing current => "/v2/received-text-messages"
-                     #                    next => "/v2/received-text-messages?older_than=last_id_in_list"
-received_texts.collection # => [] (array of ReceivedText objects)
+older_than: '8e222534-7f05-4972-86e3-17c5d9f894e2'
 ```
 
-`ReceivedText` -
+If you leave out the `older_than` argument, the client returns the most recent 250 notifications.
 
-```ruby
-received_text.id            # => uuid for the received text
-received_text.created_at    # => created_at of the received text
-received_text.content       # => content of the received text
-received_text.notify_number # => number received text was sent to
-received_text.service_id    # => service id of the received text
-received_text.user_number   # => number received text was sent from
-
-```
+The client will only return notifications that are 7 days old or less. If the notification specified in this argument is older than 7 days, the client will return an empty `collection` response.
 
-</details>
+### Response
 
-#### Arguments
+If the request to the client is successful, the client will return a `Notifications::Client::ReceivedTextCollection` object. This object is named `response` in the method example.
 
-Omit the argument Hash if you do not want to filter the results.
+You must then call either the `.links` method or the `.collection` method on this object:
 
-<details>
-<summary>
-Click here to expand for more information.
-</summary>
+|Method|Information|
+|:---|:---|
+|`response.links`|Returns a hash linking to the requested texts (limited to 250)|
+|`response.collection`|Returns an array of the required texts|
 
-##### `older_than`
-You can get the notifications older than a given `received_text.id`.
-You can omit this argument to ignore this filter.
+If you call the `collection` method on this object to return an array, you will then have to call different methods on this array to return the requested information.
```

Comment:
I think it's worth including more information about what's inside the collection array - the array contains `Notifications::Client::ReceivedText` objects and we don't document what they are or what methods they have anywhere else in the documentation.

Reply:
@klssmith same thing occurred to me earlier in the doc. Can we discuss?

Reply:
Get information from original docs readme.md